src/components/Operation.js

```diff
@@ -0,0 +1,227 @@
+/**
+ * Original file: https://github.com/Kong/swagger-ui/blob/main/src/core/components/operation.jsx
+ * @prettier
+ */
+
+import React, { PureComponent } from "react"
+import { getList, getExtensions, escapeDeepLinkPath, sanitizeUrl } from '../helpers/helpers'
+import { List } from "immutable"
+
+
+export default class Operation extends PureComponent {
+  static defaultProps = {
+    operation: null,
+    response: null,
+    request: null,
+    specPath: List(),
+    summary: ""
+  }
+
+  render() {
+    let {
+      specPath,
+      response,
+      request,
+      toggleShown,
+      onTryoutClick,
+      onCancelClick,
+      onExecute,
+      fn,
+      getComponent,
+      getConfigs,
+      specActions,
+      specSelectors,
+      authActions,
+      authSelectors,
+      oas3Actions,
+      oas3Selectors
+    } = this.props
+    let operationProps = this.props.operation
+
+    let {
+      deprecated,
+      isShown,
+      path,
+      method,
+      op,
+      tag,
+      operationId,
+      allowTryItOut,
+      displayRequestDuration,
+      tryItOutEnabled,
+      executeInProgress
+    } = operationProps.toJS()
+
+    let {
+      description,
+      externalDocs,
+      schemes
+    } = op
+
+    let operation = operationProps.getIn(["op"])
+    let responses = operation.get("responses")
+    let parameters = getList(operation, ["parameters"])
+    let operationScheme = specSelectors.operationScheme(path, method)
+    let isShownKey = ["operations", tag, operationId]
+    let extensions = getExtensions(operation)
+
+    const Responses = getComponent("responses")
+    const Parameters = getComponent( "parameters" )
+    const Execute = getComponent( "execute" )
+    const Clear = getComponent( "clear" )
+    const Collapse = getComponent( "Collapse" )
+    const Markdown = getComponent( "Markdown" )
+    const Schemes = getComponent( "schemes" )
+    const OperationServers = getComponent( "OperationServers" )
+    const OperationExt = getComponent( "OperationExt" )
+    const OperationSummary = getComponent( "OperationSummary" )
+    const Link = getComponent( "Link" )
+
+    const { showExtensions } = getConfigs()
+
+    // Merge in Live Response
+    if(responses && response && response.size > 0) {
+      let notDocumented = !responses.get(String(response.get("status"))) && !responses.get("default")
+      response = response.set("notDocumented", notDocumented)
+    }
+
+    let onChangeKey = [ path, method ] // Used to add values to _this_ operation ( indexed by path and method )
+
+    return (
+      <div
+        className={deprecated ? "opblock opblock-deprecated" : isShown ? `opblock opblock-${method} is-open` : `opblock opblock-${method}`}
+        tabIndex={0}
```

Comment:
Is that `tabIndex={0}` only change in that file or I missed something?

Reply:
Yep it's the only change :D